pycheribuild/projects/cross/cheribsd.py

```diff
@@ -148,6 +148,11 @@ def setup_config_options(cls, **kwargs):
             cls.build_tests = cls.add_bool_option("build-tests", help="Build the tests too (-DWITH_TESTS)",
                 show_help=True, default=True)
 
+        if "build_dtrace_tests" not in cls.__dict__:
```

Comment:
You probably don't need this check since it's unlikely that a subclass would define it to be always true/false